worlds/hades/Items.py

```diff
@@ -0,0 +1,422 @@
+from tokenize import group
+import typing
+
+from BaseClasses import Item, ItemClassification
+from typing import Dict
+
+class ItemData(typing.NamedTuple):
+    code: typing.Optional[int]
```

Comment:
```suggestion
from typing import Dict, NamedTuple, Optional
from BaseClasses import Item, ItemClassification

class ItemData(NamedTuple):
    code: Optional[int]
```
Would recommend not having multiple import lines from the same thing. Either import it, or import things from it.